include/GafferScene/ScenePlug.h

```diff
@@ -222,12 +223,37 @@ class GAFFERSCENE_API ScenePlug : public Gaffer::ValuePlug
 		IECore::MurmurHash setHash( const IECore::InternedString &setName ) const;
 		//@}
 
+		/// Returns true if the specified location exists. This is achieved
+		/// by querying `childNames()` at all ancestor locations, but with
+		/// significantly better performance than is achievable via the public
+		/// API alone.
+		///
+		/// > Note : It is a programming error to trigger a compute for a
+		/// > location which does not exist.
```

Comment:
Is this the best place for this comment?  I know that you mean that this is why you might use exists(), but it feels slightly weird for that comment to be here when it's OK to call exists for any location, and for anyone looking at the implementation, it's OK to trigger computes of existsPlug() for locations which don't exist.

Reply:
Good point - moved it to the top of the convenience accessors section in 7799022, and mentioned that `exists()` should be used to check where necessary.